modules/external-api/src/main/java/org/opencastproject/external/endpoint/EventsEndpoint.java

```diff
@@ -1641,8 +1641,8 @@ public Response getEventPublications(@HeaderParam("Accept") String acceptHeader,
 
   private List<JValue> getPublications(Event event, Boolean withSignedUrls, Boolean includeInternalPublication, ApiVersion requestedVersion) {
         return event.getPublications().stream()
-        .filter(publication -> ((includeInternalPublication && !requestedVersion.isSmallerThan(VERSION_1_11_0)) || EventUtils.internalChannelFilter.apply(publication)))
-        .map(p -> getPublication(p, withSignedUrls, requestedVersion))
+          .filter(publication -> ((includeInternalPublication != null && includeInternalPublication && !requestedVersion.isSmallerThan(VERSION_1_11_0)) || EventUtils.internalChannelFilter.apply(publication)))
+          .map(p -> getPublication(p, withSignedUrls, requestedVersion))
         .collect(Collectors.toList());
```

Comment:
As @KatrinIhler said, could you indent this line to be consistent with the two lines above?

Reply:
I am sorry, I thought I already did it, but actually I forgot to address this. The format should now be consistent.